base/applications/charmap_new/CMakeLists.txt

```diff
@@ -21,7 +13,12 @@ add_library(charmap MODULE
     charmap.rc)
 
 set_module_type(charmap win32gui UNICODE)
-target_link_libraries(charmap uuid wine)
+target_link_libraries(charmap uuid wine cpprt atl_classes)
+set_target_properties(charmap PROPERTIES
+    WITH_CXX_EXCEPTIONS TRUE
+    WITH_CXX_RTTI TRUE)
+# HACK: this should be enabled globally!
+target_compile_features(charmap PRIVATE "$<$<NOT:$<CXX_COMPILER_ID:MSVC>>:cxx_std_11>")
```

Comment:
CMake just won't do anything on MSVC, will it?

Reply:
> cxx_std_11
**CRINGE**

Reply:
By the way, have anybody tried to actually enable it globally?)

Reply:
In fact GCC 8.3 is at C++14 by default, so I'm not even sure this is needed at all

Reply:
Would be great if we can remove all of these or do it globally.

Reply:
Done